### ISSUE TITLE
🐛 CSS parser does not recognize multiple semicolons after a declaration

### ISSUE THREAD

**Issue by blutorange** [biomejs/biome#3836]
### Environment information

```block
CLI:
  Version:                      1.8.3
  Color support:                true

Platform:
  CPU Architecture:             x86_64
  OS:                           linux

Environment:
  BIOME_LOG_DIR:                unset
  NO_COLOR:                     unset
  TERM:                         "xterm-256color"
  JS_RUNTIME_VERSION:           "v18.19.1"
  JS_RUNTIME_NAME:              "node"
  NODE_PACKAGE_MANAGER:         "yarn/4.4.0"

Biome Configuration:
  Status:                       Loaded successfully
  Formatter disabled:           false
  Linter disabled:              false
  Organize imports disabled:    false
  VCS disabled:                 true

Workspace:
  Open Documents:               0
```

### What happened?

Biome's CSS parser raises a parser error when a CSS declaration ends with multiple semicolons, e.g.

```css
.foo {
  color: red;;
}
```

 Sorry if it was reported already, but I couldn't find an issue for that.

### Expected result

The CSS file parses correctly. The formatter could simply remove multiple semicolons. This is what prettier does

![image](https://github.com/user-attachments/assets/b1bcde4c-7dc1-4deb-a5b3-00e3421da53b)

### More info

For some context: One of my colleagues use IntelliJ to author a CSS file, and it did not show any errors. During the pre-commit hook, when biome was called to format the CSS file, it reported a syntax error in the CSS file. Took him some time to find the issue. It turned out the issue was that a declaration in the CSS file had 2 semicolons at the (`;`)

I took a look at the CSS specification. If I understand it correctly, [CSS syntax 3, 5.4.5, Consume a list of declarations](https://www.w3.org/TR/css-syntax-3/#consume-list-of-declarations) allows for multiple semicolons:

```
Repeatedly consume the next input token
  <semicolon-token>
    Do nothing.
```

Various other tools such as IntelliJ, VSCode or https://jigsaw.w3.org/css-validator/validator also accept multiple semicolons. Biome however produces a parser error (tested with 1.8.3)

![image](https://github.com/user-attachments/assets/c57770d5-4445-4297-b8d0-e597b305cf73)

According to this comment here in [declaration_or_rule_list_block.rs](https://github.com/biomejs/biome/blob/main/crates/biome_css_parser/src/syntax/block/declaration_or_rule_list_block.rs#L76-L88), the parser only seems to consider a single semicolon for now.

### Code of Conduct

- [X] I agree to follow Biome's Code of Conduct

**Comment by h-a-n-a** [comment]
I would be happy to help!

**Comment by chansuke** [comment]
@h-a-n-a 
go ahead💪

**Comment by h-a-n-a** [comment]
Sorry. I'm a little bit busy in these days and I cannot handle this at the moment. I have to unassign ;-(.
cc @chansuke
